src/pages/cheoljinJu/Main/Feed/Feed.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import Comment from '../Comment/Comment';
+import CommentForm from '../CommentForm/CommentForm';
+import './Feed.scss';
+
+class Feed extends Component {
+  state = {};
+
+  componentDidMount = () => {
+    this.setState(this.props.feed);
+  };
+
+  handleAdd = value => {
+    const comment = {
+      id: Date.now(),
+      userName: '24_Wecode',
+      comment: value,
+      isUser: true,
+      isLike: false,
+    };
+    const comments = [...this.state.comments, comment];
+    this.setState({ comments });
+  };
+
+  handleDelete = reply => {
+    const comments = this.state.comments.filter(item => reply.id !== item.id);
+    this.setState({ comments });
+  };
+
+  handleLike = reply => {
+    const comments = this.state.comments.map(item => {
+      if (item.id === reply.id) {
+        item.isLike = !item.isLike;
+        return item;
+      } else {
+        return item;
+      }
+    });
+    this.setState({ comments });
+  };
+
+  render() {
+    const { comments } = this.state;
+    const totalLike =
+      comments && comments.filter(item => item.isLike === true).length;
+    const { feed } = this.props;
+    const { profile, userName, url } = feed;
```

Comment:
- 같은 정보(feed 관련)가 props에도 있고 state 안에도 있는데, 둘 중에 한 가지만 사용하도록 해주세요! 동일한 정보를 여러군데에서 보관하고 있고, 그때그때마다 다른 곳을 참조하면 그 두 장소의 싱크를 맞춰주어야 하기 때문에 비효율적입니다. 
- 전달받은 feed 정보를 state 에 담아서 관리하고 계시니, state 를 사용해주시는게 더 좋아보입니다.
```suggestion
    const { comments, profile, userName, url } = this.state;
    const totalLike = comments && comments.filter(item => item.isLike === true).length;
```
- constructor 에서 바로 할당을 해주시면 totalLike 에서 추가적으로 조건부를 걸지 않으셔도 됩니다.
```jsx
const totalLike = comments.filter(item => item.isLike === true).length;
```

Reply:
말씀해주신대로 적용해보니 코드가 훨씬 깔금하고 일관성 있어졌습니다. 피드백 정말 감사합니다! ✨